react/components/DataLayerApolloWrapper.js

```diff
@@ -21,7 +21,11 @@ class DataLayerApolloWrapper extends Component {
       return
     }
 
-    this.props.set(data)
+    if (Array.isArray(data)) {
+      data.map(element => this.props.set(element))
```

Comment:
use `forEach` and pass the `this.props.set` reference as a direct argument (e.g. `data.forEach(this.props.set)`)